typescript.js

```diff
@@ -361,6 +361,9 @@ module.exports = {
         'quotes': 'off', // covered by @typescript-eslint/quotes
         'semi': 'off', // covered by @typescript-eslint/semi
         'space-before-function-paren': 'off', // covered by @typescript-eslint/space-before-function-paren
+        'no-undef': 'off', // off because variables in d.ts files will be shown as error otherwise
+        'no-unused-vars': 'off', // covered by @typescript-eslint/no-unused-vars
+        'no-var': 'off', // covered by @typescript-eslint/no-var
```

Comment:
There is no rule `@typescript-eslint/no-var`
I would simply turn that off at the overrides section for `d.ts` files, what do you mean?

Reply:
Somewhere around line 370.